scico/linop/_circconv.py

```diff
@@ -124,7 +127,18 @@ def __init__(
             output_dtype = result_type(h.dtype, input_dtype)
 
             if self.h_center is not None:
-                offset = -self.h_center
+                if isinstance(self.h_center, DeviceArray):
+                    offset = -self.h_center
+                else:
+                    # support float or int values for h_center
+                    if isinstance(self.h_center, (float, int)):
+                        offset = -snp.array(
```

Comment:
formatting looks slightly odd here but I assume that's black's doing

Reply:
Do you mean the relatively deeply nested `if`/`else` structure? That's just a natural consequence of the actual logic here.